frontend/src/app/libs/common-components/components/event-card/models/event-card-role.ts

```diff
@@ -0,0 +1,7 @@
+export enum EventCardRole {
+	Participant = 'Participant',
+	Mentor = 'Mentor',
+	HR = 'HR',
+	Manager = 'Manager',
+	Admin = 'Admin'
```

Comment:
Admin role can be removed, since it's global application role, not applicable for events :)